packages/ckeditor5-list/lang/contexts.json

```diff
@@ -24,6 +24,7 @@
 	"Upper-latin": "The tooltip text of the button that toggles the \"upper–latin\" list style.",
 	"List properties": "The label of the button that toggles the visibility of additional numbered list property UI fields.",
 	"Start at": "The label of the input allowing to change the start index of a numbered list.",
+	"Invalid start index value.": "The error message displayed when the numbered list is not a valid number",
```

Comment:
There is no `.` at the end of the message.

Reply:
I don't get it, I did the same as the other existing example below:
<img width="649" alt="image" src="https://github.com/ckeditor/ckeditor5/assets/85623325/95979986-5333-4957-8810-1cf6e427d594">

Reply:
Please explain what do you expect here

Reply:
okay, not the id but the message, now I get you I think:)